docs/source/release_notes.rst

```diff
@@ -11,6 +11,7 @@ Release Notes
         * Added percent-better-than-baseline for all objectives to automl.results :pr:`1244`
         * Added ``HighVarianceCVDataCheck`` and replaced synonymous warning in ``AutoMLSearch`` :pr:`1254`
         * Added `PCA Transformer` component for dimensionality reduction :pr:`1270`
+        * Updated ``AutoMLSearch`` to support Woodwork data structures :pr:`1299`
```

Comment:
Could you please also add an entry to breaking changes mentioning that a warning is generated when pandas types are provided? I think its good to call out that change in behavior.